src/app/pages/apps/components/app-detail-view/app-detail-view.component.html

```diff
@@ -63,9 +63,14 @@ <h5 class="catalog-header">{{ 'Official Catalog' | translate }}</h5>
           </ng-template>
```

Comment:
Middleware recently updated API to have tags, worth updating here from category https://github.com/truenas/middleware/pull/11198